pebbles/logger.py

```diff
@@ -16,17 +15,16 @@ def __init__(self, api_base_url, instance_id, token, ssl_verify=True):
         auth = base64.encodestring('%s:%s' % (token, '')).replace('\n', '')
         self.ssl_verify = ssl_verify
         self.headers = {
-            'Content-type': 'application/x-www-form-urlencoded',
             'Accept': 'text/plain',
             'Authorization': 'Basic %s' % auth}
         self.url = '%s/instances/%s/logs' % (api_base_url, instance_id)
 
     def emit(self, record):
-        log_record_json = self.format(record)
-        payload = {'log_record_json': log_record_json}
+        log_record = self.format(record)
```

Comment:
what exactly changes here? you changed the way you use requests.patch?

Reply:
Yes, there's a parameter called 'json' which allows sending json objects directly , so there's no need of dumping json objects to strings and loading back